python/pyspark/ml/tests.py

```diff
@@ -1355,7 +1370,7 @@ def test_java_params(self):
             for name, cls in inspect.getmembers(module, inspect.isclass):
                 if not name.endswith('Model') and issubclass(cls, JavaParams)\
                         and not inspect.isabstract(cls):
-                    self.check_params(cls())
+                    ParamTests.check_params(self, cls(), check_params_exist=False)
```

Comment:
Setting `check_params_exist` to True will uncover any params that exist in Java but not in Python